x-pack/test/security_solution_cypress/cypress/cypress_serverless.config.ts

```diff
@@ -26,6 +26,7 @@ export default defineCypressConfig({
     grepTags: '@serverless --@brokenInServerless --@skipInServerless',
   },
   e2e: {
+    experimentalCspAllowList: ['default-src', 'script-src', 'script-src-elem'],
```

Comment:
Exactly before the env field we should be adding the reporters. this is mandatory for the junit xml output to be created and then the `junit:merge` command to run. 
```
reporter: '../../../node_modules/cypress-multi-reporters',
  reporterOptions: {
    configFile: './cypress/reporter_config.json',
  },
```

Reply:
We just use the reporters for the headless-mode, we don't need it for the visual, and we use this configuration just for visual executions.